sqlitedict.py

```diff
@@ -552,6 +573,28 @@ def close(self, force=False):
             self.select_one('--close--')
             self.join()
 
+    def wait_for_initialization(self):
```

Comment:
I think it's worth making this thing explicitly internal, so people don't touch it (or at least expect bad things to happen if they do).

```suggestion
    def _wait_for_initialization(self):
```